### ISSUE TITLE
Escaped strings in JSON not escaped in POT

### ISSUE THREAD

**Issue by jonathancochran** [rubencosta/react-intl-gettext#4]
Great library!

There's an issue though when generating the POT file where escaped strings in json do not get escaped in the POT file. 

For example, this:
```
{
  "id": "my.string.id",
  "defaultMessage": "You should <a href=\"http://www.test.com/\" target=\"_blank\">click here</a>",
  "description": "Amazing link"
},
```

Produces:
``` 
#. Amazing link
# en.json
msgctxt "my.string.id"
msgid "You should <a href="http://www.test.com" target="_blank">click here</a>"
msgstr ""
```

**Comment by rubencosta** [comment]
Thanks for noticing this. I just released `5.1.2` that should escape the strings in POT files.

**Comment by jonathancochran** [comment]
Great, thank you @rubencosta!!
